valohai_yaml/objs/parameter.py

```diff
@@ -199,11 +203,16 @@ def lint(
         lint_result: LintResult,
         context: LintContext,
     ) -> None:
+        prefix = f'Step {context["step"].name}, parameter {self.name}'
+        self.lint_flags(prefix, lint_result)
+        self.lint_default(prefix, lint_result)
```

Comment:
What's the reason for this change/refactoring, by the way? I don't see it described in the commit message or PR description...